pyvo/registry/regtap.py

```diff
@@ -122,6 +122,36 @@ def search(*constraints: rtcons.Constraint, includeaux: bool = False, maxrec: in
     *constraints : `rtcons.Constraint` instances
         The constraints (keywords to match, positions to cover, ...)
         that the returned records need to satisfy.
+        The accepted constraints are:
+        
+        - keywords: one or more freetext words, mached in the title, 
```

Comment:
Add an indent here and for all the other items in the list to make the rendering nicer.

```suggestion
          - keywords: one or more freetext words, mached in the title, 
```

Reply:
yes. The whitepace linting issue is still there, as well as some extra whitespace with the rendering (https://pyvo--426.org.readthedocs.build/en/426/api/pyvo.registry.regtap.search.html#pyvo.registry.regtap.search), I'll just add a commit that fixes all those rather than going back and forth with it.